guides/source/testing.md

```diff
@@ -445,7 +445,7 @@ By default test failures and errors are reported inline during a run.
 
 minitest options:
     -h, --help                       Display this help.
-        --no-plugins                 Bypass minitest plugin auto-loading (or set $MT_NO_PLUGINS).
+        --no-plugins                 Bypass minitest plugin autoloading (or set $MT_NO_PLUGINS).
```

Comment:
This output is printed by minitest.

https://github.com/seattlerb/minitest/blob/3c6576a51f4e266996e3459d7a0dd054eb4c87f7/lib/minitest.rb#L183

Reply:
Thanks! I reverted this line.